### ISSUE TITLE
useful README

### ISSUE THREAD

**Issue by ppeczek** [KlubJagiellonski/Politikon#226]
Remove Mac OS X part. Make README ASAP (as short as possible). Think about creating Wiki

**Comment by ppeczek** [comment]
@JulianVolodia , you mentioned that you're going to make README useful. May I assign you to this issue?

**Comment by JulianVolodia** [comment]
yep. today will be PR - conclusion of my and Wojciech Raczek's work from 12 days ago ;/ sorry for latency...

**Comment by ppeczek** [comment]
No problem, can't wait for PR :+1:
